packages/babel-standalone/Gulpfile.js

```diff
@@ -61,14 +63,15 @@ function webpackBuild(filename, libraryName, version) {
     plugins: [
       new webpack.DefinePlugin({
         "process.env.NODE_ENV": '"production"',
-        BABEL_VERSION:
-          JSON.stringify(require("babel-core/package.json").version),
+        BABEL_VERSION: JSON.stringify(
+          require("babel-core/package.json").version,
```

Comment:
Why wouldn't it be the babel-core version being compiled? If we're in babel-standalone, wouldn't `require('babel-core')` give us the babel-core that's been symlinked via Lerna?